lib/Inst/Inst.cpp

```diff
@@ -173,6 +176,10 @@ std::string ReplacementContext::printInst(Inst *I, llvm::raw_ostream &Out,
           Out << " (signBits=" << I->NumSignBits << ")";
       }
       Out << OpsSS.str();
+
+      if (OrigI && OrigI->DepsWithExternalUses.find(I) != OrigI->DepsWithExternalUses.end())
```

Comment:
so, for example, here you check OrigI against nullptr but didn't you already filter out the case where OrigI == nullptr? again, I think there must be a nicer way to say all this